app/src/main/java/de/tum/in/tumcampusapp/api/app/TUMCabeClient.java

```diff
@@ -84,15 +79,19 @@
     static final String API_KINOS = "kino/";
     static final String API_CARD = "cards/";
     static final String API_NEWS = "news/";
-
+    private static final String API_HOSTNAME = Const.API_HOSTNAME;
+    private static final String API_BASEURL = "/Api/";
+    private static final String API_CHAT = "chat/";
+    static final String API_CHAT_ROOMS = API_CHAT + "rooms/";
+    static final String API_CHAT_MEMBERS = API_CHAT + "members/";
```

Comment:
The usual best practice is to have `public` / package scoped variables before `private` variables, since `private` variables shouldn't be visible :)

Anyways, the auto-formatter moved those around, so I didn't really give it a thought :man_shrugging: